### PR TITLE
code for Hanee's blog post

### PR DESCRIPTION
**Pull request by nitbix**

### CODE REVIEW

**Comment by nitbix** [comment]
@aturley when you merge this, please "squash and merge" - I don't want my twitter keys to appear on github ;)

File: twitter_wallaroo_app.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+class HashtagFinder(object):
+    def name(self):
+        return "HashtagFinder"
+
+    def compute_multi(self, data):
+        return [word.strip() for word in data.split() if word[0] == '#']
+
+
+class HashtagCounts(object):
+
```
Comment (nisanharamati) [inline twitter_wallaroo_app.py]: no newline here

File: twitter_wallaroo_app.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import sys
+print sys.path
+
+import struct
+import wallaroo
+import pandas as pd
+
```
Comment (nisanharamati) [inline twitter_wallaroo_app.py]: should be two linebreaks

File: socket_receiver.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import socket
+import requests
+import ast
+
```
Comment (nisanharamati) [inline socket_receiver.py]: should be two linebreaks

File: twitter_client.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import socket
+import sys
+import requests
+import requests_oauthlib
+import json
+
```
Comment (nisanharamati) [inline twitter_client.py]: two linebreaks

File: twitter_wallaroo_app.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
+import sys
+print sys.path
+
```
Comment (nisanharamati) [inline twitter_wallaroo_app.py]: two linebreaks

**Comment by nisanharamati** [comment]
You need to add a .gitignore and remove the `.pyc` file

**Pull request merged**